_drafts/2024-05-18-get-started-with-aws-tf-01.md

```diff
@@ -0,0 +1,254 @@
+---
+layout: post
+title: "Getting Started with AWS and Terraform: 01 - Creating an EC2 Intance"
+date: 2024-05-17 16:00:00 +0800
+categories: aws terraform infratructure IaC
+description: In this multi-part blog series we will learn how to create a modern enterprise ready web service in AWS using Terraform. In this first post we will create an EC2 instance to host our web service in AWS using Terraform
+---
+
+# Introduction
```

Comment:
I think it'd be worth outlining some of the assumed knowledge you'd want readers to have. It'll help you figure out what level of detail to include (as referenced in above comments) but also provide readers with a quick way to figure out if they will be able to consume this information.

My (opinionated) take on assumed knowledge would be something like the below but feel free to customise this as you want.

> It is assumed you have basic, entry-level knowledge of the following:
> - What Infrastructure as Code (IaC) is (eg Terraform, Pulumi, CloudFormation)
> - What a cloud infrastructure provider is (eg AWS, Google Cloud, Azure)
> - How to run basic commands in a terminal
> - How to use a code editor or IDE
> - Basic knowledge of a programming language

You could then optionally provide some links to some docs that introduce some of these concepts if readers are unfamiliar or just need a quick reminder.